.gitignore

```diff
@@ -32,6 +32,7 @@ doc/locale/
 tests/.coverage
 tests/build/
 tests/test-server.lock
+tests/locks/test-server.*.lock
```

Comment:
Arg.  Straight after I say that: OK; one further change.  We don't need these `.gitignore` entries any longer.